### DISCUSSION TITLE
ctexbook.cls 问题

### DISCUSSION THREAD

**Discussion by obstacle213** [sjtug/SJTUThesis#833] [提问]
之前操作都正常，后面出现与Miktex相关的问题，然后我把texlive相关的卸载重装
编译后出现 Key 'ctex/option/zihao' accepts only a fixed set of choices. }
[main.log](https://github.com/sjtug/SJTUThesis/files/9420798/main.log)

TeX Live，windows

**Comment by skyzh** [comment]
最新版应该已经修复了，尝试使用 master 分支再试一次？

**Reply by obstacle213** [reply]
是指下载最新的模板尝试吗
